### ISSUE TITLE
Release IPA

### ISSUE THREAD

**Issue by d-513** [Inokinoki/kdeconnect-ios#5]
Altstore could be used to side load the app when it’s not yet on app store

**Comment by Inokinoki** [comment]
Right, I'll work on this version for a backward compatibility on iOS 10+. And do an ipa release in this repo

**Comment by d-513** [comment]
https://testflight.apple.com/join/vxCluwBF exists now official from kde devs

**Comment by Inokinoki** [comment]
Yes, it is our official port with SwiftUI which needs iOS 14 or above. It is perfect if that already fits your requirement.

I'll meanwhile keep maintaining this project for the iOS 10+ devices.
